docs/sources/flow/reference/components/otelcol.receiver.jaeger.md

```diff
@@ -0,0 +1,289 @@
+---
```

Comment:
The bulk of the content here is copied from `otelcol.receiver.otlp`. Hopefully once #2341 is in place we can cut down on the review load.